packages/rest-specs/specs/authentication/api-key/main.tsp

```diff
@@ -0,0 +1,39 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
```

Comment:
I think all of those need to change to the typespec package name

Reply:
Done